app/lib/logger_error_subscriber.rb

```diff
@@ -0,0 +1,7 @@
+# This subscriber will log exceptions to stdout, when using
+# new Rails error reporting API (i.e. `Rails.error.handle`, etc.)
+class LoggerErrorSubscriber
+  def report(error, *)
+    Rails.logger.error [error, error.backtrace[0]].join("\n")
```

Comment:
Could we use `backtrace.first` here? Makes it a little easier to parse maybe

Reply:
Changed in next PR 👍